core/src/test/java/com/scalar/db/transaction/consensuscommit/ParallelExecutorTest.java

```diff
@@ -693,13 +695,24 @@ public void executeTasks_ParallelTrueAndStopOnErrorTrue_ExceptionThrown_ShouldSt
 
     List<ParallelExecutorTask> mixedTasks = Arrays.asList(failingTask1, failingTask2, task);
 
-    // Act Assert
-    assertThatThrownBy(
+    // Act
+    Exception exception =
+        catchException(
             () ->
                 parallelExecutor.executeTasks(
-                    mixedTasks, parallel, noWait, stopOnError, "test", TX_ID))
-        .isEqualTo(executionException1)
-        .hasSuppressedException(executionException2);
+                    mixedTasks, parallel, noWait, stopOnError, "test", TX_ID));
+
+    // Assert
+    assertThat(exception)
+        .satisfiesAnyOf(
+            e ->
```

Comment:
The updated code checks two patterns for the exception, but the original code checks only one pattern, so isn't this a bug fix rather than an improvement?

Reply:
You are right. I've changed the label. Thanks!